components/automate-builder-api-proxy/habitat/plan.sh

```diff
@@ -0,0 +1,55 @@
+#shellcheck disable=SC2034
+#shellcheck disable=SC2039
+#shellcheck disable=SC2154
+
+pkg_name="automate-builder-api-proxy"
+pkg_origin="chef"
+pkg_description="Wrapper package for habitat/builder-api-proxy"
+pkg_version="0.1.0"
+pkg_maintainer="Chef Software Inc. <support@chef.io>"
+pkg_license=("Chef-MLSA")
+pkg_upstream_url="https://www.chef.io/automate"
+pkg_svc_user="root"
+pkg_deps=(
+  core/runit
+  chef/mlsa
+  "${local_platform_tools_origin:-chef}/automate-platform-tools"
+  # We need to pin here to get a build from unstable
```

Comment:
Is this just to get the build that has your fixes in it?

Reply:
we'll want to auto-bump this